e2e/steps_file.js

```diff
@@ -173,6 +173,18 @@ module.exports = function () {
       return `${selector}//*[contains(@class,"complex-panel-simple-field") and .//th/span[text()="${fieldName}"]]`;
     },
 
+    organisationTabFieldSelector(pathToField) {
```

Comment:
Maybe it could be a little task after but it would be nice if all these tab assertion related methods were in their own helper

Reply:
will do :)